packages/app/package.json

```
@@ -276,7 +276,7 @@
     "@types/semver": "6.2.0",
     "@types/socket.io-client": "^1.4.32",
     "@types/stripe-v3": "^3.1.7",
-    "@types/styled-components": "^4.4.2",
+    "@types/styled-components": "^5.1.0",
```

You only updated `styled-component` types, so these can be reverted I think? 🤔

```suggestion
    "@types/styled-components": "^4.4.2",
```

---

The 4.4.2 version is not compatible with TS 3.9 :-( But we updated to styled-components 5.1.0 instead 😄